UnitTests/Views/ShortcutTests.cs

```diff
@@ -314,4 +314,259 @@ public void Focus_CanFocus_CommandView_Add_Tracks ()
         Assert.False (shortcut.CanFocus);
         Assert.True (shortcut.CommandView.CanFocus);
     }
+
+    [Theory]
+    //  0123456789
+    // " C  0  A "
+    [InlineData (-1, 0)]
```

Comment:
How about, instead of all these inlines:

```csharp
    [Theory]
    [CombinatorialData]
    [InlineData (-1, 0)]
    [InlineData (9, 0)]
    [AutoInitShutdown]
    public void MouseClick_Fires_Accept ([CombinatorialRange (0, 8)] int x, [CombinatorialValues (1)] int expectedAccept)

```

Or similar.

InlineData can still be used for one-off cases, as shown, and are explicit cases beyond the "combinatorial" cases, but do get de-duplicated if you accidentally specify any dupes.

Don't bother if you don't want to, though.

There are other things about the tests that are more important, but that's so far down my list right now. 😅